hermes/src/ontology/mod.rs

```diff
@@ -22,8 +12,43 @@ pub use self::nlu::*;
 pub use self::tts::*;
 pub use self::vad::*;
 
+pub mod asr;
+pub mod audio_server;
+pub mod dialogue;
+pub mod hotword;
+pub mod injection;
+pub mod nlu;
+pub mod tts;
+pub mod vad;
+
 pub trait HermesMessage<'de>: fmt::Debug + Deserialize<'de> + Serialize {}
 
+#[derive(Serialize, Deserialize, PartialEq, PartialOrd, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub enum SnipsComponent {
```

Comment:
call it simply Component

Reply:
In fact we don't need it anymore. We will do another PR to change all current component string by this enum